src/ast/mod.rs

```diff
@@ -141,6 +143,21 @@ impl Ident {
             quote_style: Some(quote),
         }
     }
+
+    /// Returns the defined `Keyword` enum for this identifier if it is a keyword.
```

Comment:
```suggestion
    /// If this identifier is also a keyword, return the corresponding [`Keyword`].
    /// 
    /// For example even though `AVRO` is a keyword, it can also be used as an
    /// identifier for a column, such as `SELECT avro FROM my_table`.
```